README.md

```diff
@@ -42,13 +42,10 @@ fastify.register(require('fastify-swagger'), {
     ],
     definitions: {
       User: {
-        $id: 'User',
         type: 'object',
         required: ['id', 'email'],
         properties: {
           id: { type: 'string', format: 'uuid' },
-          firstName: { type: 'string', nullable: true },
-          lastName: { type: 'string', nullable: true },
```

Comment:
Can you just remove `nullable: true` and not the fields?

Reply:
Yep!